lib/content-services/src/lib/content-node-selector/content-node-selector.component-data.interface.ts

```diff
@@ -27,6 +27,7 @@ export interface ContentNodeSelectorComponentData {
     rowFilter?: any;
     where?: string;
     imageResolver?: any;
+    selectionMode: string;
```

Comment:
```suggestion
    selectionMode: 'single' | 'multiple';
```

in this case we will get maximum value safety and validation